src/components/Waveform.js

```diff
@@ -48,7 +48,16 @@ class Waveform extends Component {
       });
       hls.on(Hls.Events.ERROR, function(event, data) {
         if (data.fatal) {
```

Comment:
Could this conditional be removed too?  It doesn't look like it is really doing anything if you have that default case in the switch.

Reply:
According to `Hls.js` documentation, if an error is not fatal (for an example: the loading times out) `Hls.js` will try to recover it. If it's fatal, then only an action is required to recover it. Do you think maybe we should remove it?

Reply:
:+1: I'm fine with this then.